examples/java/Swing.java

```diff
@@ -62,10 +62,10 @@ public void paint(final Graphics g) {
 		g2d.fillRect(0, 0, getWidth(), getHeight());
 
 		// interpolate a sequence of bezier curves
-		final List<Float> toInterpolate = new ArrayList<>();
+		final List<Double> toInterpolate = new ArrayList<>();
 		points.forEach(p -> {
-			toInterpolate.add((float) p.getX());
-			toInterpolate.add((float) p.getY());
+			toInterpolate.add((double) p.getX());
```

Comment:
The cast to double can be removed :).

Reply:
And done!  (I'm not familiar with Java, to be honest... 😛 )